### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by meetmangukiya**
Not sure if it is needed, but something I thought should be changed. It was a single word, so didn't file an issue and all. :laughing:

### CODE REVIEW

**Comment by meetmangukiya** [comment]
@atjonathan ^^

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ gulp
 We welcome any pull requests or improvements to our code you might have. Please follow [these steps](https://github.com/extramundane/extramundane.github.io/blob/master/.github/CONTRIBUTING.md) to contribute.
 
 #### SASS Structure
-If you are unfamiliar with the SASS Syntax take a look at their [website](http://sass-lang.com/guide).
+If you are not familiar with the SASS Syntax take a look at their [website](http://sass-lang.com/guide).
```
Comment (atjonathan) [inline README.md]: ~~`If you are not familiar with the SASS Syntax take a look at their [website](http://sass-lang.com/guide).`~~ If you are not familiar with the SASS Syntax, take a look at their [website](http://sass-lang.com/guide).

Comment (meetmangukiya) [inline README.md]: :+1:

Comment (meetmangukiya) [inline README.md]: Squashed the commits too. Do you want a change in shortlog or description of commit ?

**Pull request merged**